app/api/v1/blueprints/validator.py

```diff
@@ -8,18 +8,20 @@ def __init__():
     def validate_user(user_object):
         for key, value in user_object.items():
             # ensure keys have values
-            if not value :
-                response = {
-                    'status': 400,
-                    'error': 'Please provide your {}'.format(key)
-                }
-                return response
-            if not value.strip():
-                response = {
-                    'status': 400,
-                    'error': 'Please provide a valid {}'.format(key)
-                }
-                return response
+            if not type(value) == bool:
```

Comment:
This seems to fail because not all **keys** are of type bool in your user_onject. For example, **firstname** is of type **string**

Reply:
I have updated and tested the endpoints to handle boolean  data well